training/src/anemoi/training/losses/base.py

```diff
@@ -110,11 +118,42 @@ def scale(
 
         return x[subset_indices] * scaler[subset_indices]
 
-    def reduce(self, out: torch.Tensor, squash: bool = True) -> torch.Tensor:
+    def reduce(
+        self,
+        out: torch.Tensor,
+        squash: bool = True,
+    ) -> torch.Tensor:
+        """
+        Reduce the out of the loss.
+
+        If `squash` is True, the last dimension is averaged.
+
+        Irrespective of `squash`, the output is reduced over the
+        batch, ensemble and grid dimensions.
+
+        Parameters
+        ----------
+        out : torch.Tensor
+            Difference tensor, of shape TensorDim
+        squash : bool, optional
+            Whether to squash the variable dimension, by default True
+
+        Returns
+        -------
+        torch.Tensor
+            Reduced output tensor
+        """
         if squash:
-            out = self.avg_function(out, dim=-1)
+            out = self.avg_function(out, dim=TensorDim.VARIABLE)
 
-        return self.sum_function(out, dim=(0, 1, 2))
+        grid_summed = self.sum_function(out, dim=(TensorDim.GRID))
```

Comment:
Could you add a comment to explain that this is a sum because the normalisation is handled in the GRID weighting?